pkgs/development/python-modules/lox/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+
+  pathos,
+  pip,
+  pytestCheckHook,
+  pytest-benchmark,
+  pytest-mock,
+  pytest-runner,
+  tqdm,
+}:
+
+buildPythonPackage rec {
+  pname = "lox";
+  version = "0.12.0";
+
+  src = fetchFromGitHub {
+    owner = "BrianPugh";
+    repo = "lox";
+    rev = "v${version}";
+    hash = "sha256-Iv3ZdfsvFLU6lhlH1n+eQ+KIrXESsnC1S2lVFnKFV08=";
+  };
+
+  propagatedBuildInputs = [ pathos ];
```

Comment:
```suggestion
  dependencies = [ pathos ];
```

Reply:
What is the difference here? Is propagatedBuildInputs deprecated? Or is there a difference in meaning between them that causes this to be preferred?

Reply:
Yes, see the documentation: https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md

Reply:
I'm not sure which of those you are saying "yes" to. The link you provide no longer brings up propagatedBuildInputs except to say "see the propagatedBuildInputs section" which appears to be missing. So I'm guessing that it is now deprecated in favor of dependencies?